addons/website_sale_wishlist/static/src/js/website_sale_wishlist.js

```diff
@@ -191,8 +194,8 @@ publicWidget.registry.ProductWishlist = publicWidget.Widget.extend(VariantMixin,
     /**
      * @private
      */
-    _redirectNoWish: function () {
-        window.location.href = '/shop/cart';
+    _redirectNoWish: function (redirect_url='/shop/cart') {
```

Comment:
```suggestion
    _redirectNoWish(redirect_url='/shop/cart') {
```

Suggestion: It might be good time to also update syntax to ES6 for this function since we are changing this line :)